### PR TITLE
disallow core CSS pages in verbatim tag

### PR DESCRIPTION
**Pull request by owend**

### CODE REVIEW

File: extensions/3rdparty/Verbatim/Verbatim.php [hunk 1]
```diff
@@ -18,5 +18,14 @@ function wfVerbatimExtension( $parser ) {
 
 // The callback function for converting the input text to HTML output
 function renderVerbatim( $input ) {
+	global $wgEditInterfaceWhitelist;
+
+	// Begin wikia change
+	if (is_array($wgEditInterfaceWhitelist) and in_array(trim($input), $wgEditInterfaceWhitelist)) {
```
Comment (wladekb) [inline extensions/3rdparty/Verbatim/Verbatim.php]: just in case I'd add !empty(...) so if the variable is not defined warning won't be emitted

Comment (Grunny) [inline extensions/3rdparty/Verbatim/Verbatim.php]: Probably need to check `ucfirst` on the input as MW messages are case insensitive on the first letter.

**Comment by wladekb** [comment]
lgtm apart from a small comment that might be addressed later and shouldn't be treated as a blocker

**Comment by Grunny** [comment]
LGTM once the `ucfirst` check is in place (otherwise the check can be bypassed).

**Comment by wladekb** [comment]
@Grunny good catch :+1:

**Pull request merged**